packages/enhanced-img/package.json

```
@@ -45,9 +45,13 @@
 		"@types/node": "^18.19.3",
 		"estree-walker": "^3.0.3",
 		"rollup": "^4.14.2",
-		"svelte": "^4.2.10",
+		"svelte": "^4.2.14",
 		"typescript": "^5.3.3",
 		"vite": "^5.2.8",
 		"vitest": "^1.6.0"
+	},
+	"peerDependencies": {
+		"svelte": "^4.2.14",
+		"vite": "^5.2.8"
```

you should allow older versions of these packages
```suggestion
		"svelte": "^4.0.0 || ^5.0.0-next.0",
		"vite": "^5.0.0"
```